backend/Api/Projects/ProjectController.cs

```diff
@@ -41,6 +42,34 @@ public ProjectController(ApplicationContext context, IMemoryCache cache)
             .ToList();
     }
 
+    [HttpPut]
+    [Route("updateState")]
+    public ActionResult<List<ConsultantReadModel>> Put([FromRoute] string orgUrlKey, [FromBody] UpdateProjectWriteModel projectWriteModel)
+    {
+        var selectedOrgId = _context.Organization.SingleOrDefault(org => org.UrlKey == orgUrlKey);
+        var engagement = _context.Project.Include(p=>p.Consultants).SingleOrDefault(p => p.Id == projectWriteModel.EngagementId);
+        if (selectedOrgId is null || engagement is null) return BadRequest();
```

Comment:
We should agree on some kind of type of response on BadRequest("my error here").

For now I've just used string other places.

I would want to use a snackbar to pop up if an error occurs, but thats for later ofc

Reply:
But its ok for now 👍